test/util/service_context.go

```diff
@@ -3,21 +3,16 @@ package util
 import (
 	"testing"
 
-	"github.com/codeready-toolchain/registration-service/pkg/kubeclient"
+	"github.com/codeready-toolchain/registration-service/pkg/namespaced"
 	"github.com/codeready-toolchain/registration-service/pkg/server"
 	"github.com/codeready-toolchain/registration-service/test/fake"
 	commontest "github.com/codeready-toolchain/toolchain-common/pkg/test"
-	"github.com/stretchr/testify/require"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
-func NewMemberClusterServiceContext(t *testing.T, cl client.Client) fake.MemberClusterServiceContext {
-	crtClient, err := kubeclient.NewCRTRESTClient(cl, commontest.HostOperatorNs)
-	require.NoError(t, err)
-	application, err := server.NewInClusterApplication(cl, commontest.HostOperatorNs)
-	require.NoError(t, err)
+func NewMemberClusterServiceContext(_ *testing.T, cl client.Client) fake.MemberClusterServiceContext {
 	return fake.MemberClusterServiceContext{
-		CrtClient: crtClient,
-		Svcs:      application,
+		NamespacedClient: namespaced.NewClient(cl, commontest.HostOperatorNs),
+		Svcs:             server.NewInClusterApplication(cl, commontest.HostOperatorNs),
 	}
 }
```

Comment:
This will go away in one of the following PRs. Just wanted to minimize the number of changes so I left it here for now.